rest/attachment_test.go

```diff
@@ -2990,20 +2997,19 @@ func TestCBLRevposHandling(t *testing.T) {
 	err = json.Unmarshal(doc2Bytes, &doc2Body)
 	assert.NoError(t, err)
 
-	err = btc.StartOneshotPull()
```

Comment:
I wonder if we should also remove these methods on the client (and force future tests to use `btc.DefaultCollection().StartOneShotPull()` if that's what they're actually intending to do)